compiler/src/iree/compiler/Dialect/Stream/IR/test/async_folding.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --split-input-file --canonicalize %s | iree-opt --split-input-file | FileCheck %s
+// RUN: iree-opt --split-input-file --canonicalize=test-convergence=true %s | iree-opt --split-input-file | FileCheck %s
```

Comment:
Yeah, I only noticed this because iree-util-apply-patterns has it on by default. Maybe we can just switch all tests to use that? (it would make us a bit less vanilla MLIR though)

Reply:
I think that's fine / better to ferret out such cases :)